modules/core/shared/src/main/scala/lucuma/core/model/SiderealTracking.scala

```diff
@@ -50,8 +51,8 @@ final case class SiderealTracking(
   parallax:        Option[Parallax]
 ) {
 
-  def at(i: Instant): Option[Coordinates] =
-    plusYears(epoch.untilInstant(i))
+  def at(i: Timestamp): Option[Coordinates] =
```

Comment:
We are using `Timestamp` in `Ephemeris`. Do we want it here too?

Reply:
This value is not saved in the DB. Maybe we should keep `Instant` to simplify the API?

Reply:
I suppose the `Tracking` interface could provide an `atTimestamp` that calls the instant version.  Otoh, requiring the caller who has a `Timestamp` for some reason to do `at(t.toInstant)` isn't so bad?

Reply:
I want that both sidereal and non-sidereals have the same interface. my impression is that it should be `Timestamp` rather than instant, is that right?

Reply:
As @rpiaggio pointed out, this isn't about a time that is kept in the database.  `Timestamp` is more about being able to round-trip values.  I think I would tend to keep the coordinate lookup using `Instant`.  Non-sidereal ephemeris data will be stored at microsecond precision but we should be able to estimate coordinates any `Instant` covered by that data.  If you want to provide both though (and just have the timestamp version convert itself to an instant and call the instant version), that seems fine to me.

Reply:
ok, that sounds good. I learned you can't add another `at(i: Instant)` because `Timestamp` is an opaque but we could do `atTimestamp` or just let it off